libs/SoloPlay/Functions/MiscOverrides.js

```diff
@@ -473,9 +473,9 @@ Misc.checkSocketables = function () {
 		let sockets = items[i].getStat(sdk.stats.NumSockets);
 
 		switch (items[i].quality) {
-		case sdk.itemquality.Magic:
-		case sdk.itemquality.Rare:
-		case sdk.itemquality.Crafted:
+		case item.magic:
```

Comment:
This case structure needs to be reverted back, it's checking the numerical value instead of the boolean one

Reply:
done